core/src/documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,121 @@
+        </p>
+        <p>
+            Launch your VSCode and Bring up the Extensions view by clicking on the <b>Extensions</b> icon in the Activity Bar
+            on the side of VS Code or  <b>View > Extensions</b> then write the name of extension <b>Java Extension Pack</b> and 
+            then clique on install link to install all extension for Pack.
+        </p>
+        <figure>
+            <a href="images/VSCode-1.png"><img src="images/VSCode-1.png" alt="Installing Java Extension Pack to VSCode."></a>
+	    <figcaption>Installing Java Extension Pack to VSCode.</figcaption>            
+        </figure>
+
+        <h2>Open the source code folder</h2>
+        <p>
+            The Spark source code is using an Apache Maven project structure. As VSCode (with Maven Extension) recognizes this structure,
+            opening the source code into an Vscode project is very easy.
+        </p>
+        <p>
+            Select the 'Open' option -- <b>file > open</b> and choose your spark source code folder then clique on the open button. 
+            This will quickly import the Spark source code into  VSCode .
+        </p>
+
+        <figure>
+            <a href="images/VSCode-2.png"><img src="images/VSCode-2.png" alt="Use the 'Open' functionality of VSCode."></a>
+            <a href="images/VSCode-3.png"><img src="images/VSCode-3.png" alt="Select the folder that will open in our VSCode."></a>
+            <a href="images/VSCode-4.png"><img src="images/VSCode-4.png" alt="Overview of project in Visual Studio Code"></a>
+	    <figcaption>Opening Spark Source folder on VSCode.</figcaption>            
+        </figure>
+
+        <p>
+            VSCode will automatically recognize and process the Maven structure of Spark (pom.xml file),and it will download automatically all dependancies.
+        </p>
+	    
+        <h2>Run Spark from VSCode</h2>
+        <p>
+            When developing, it is often useful to run the code straight from your IDE. The following will explain how
+            you can start a Spark instance from the code that is in the VSCode workspace.
+        </p>
+
+        <figure>
+            <a href="images/VSCode-5.png"><img src="images/VSCode-5.png" alt="Launch the 'Startup' class as a Java application."></a>
+            <a href="images/VSCode-6.png"><img src="images/VSCode-6.png" alt="The Spark login screen, after launching the 'Startup' class."></a>
+            <figcaption>Run Spark straight from the source code.</figcaption>
+        </figure>
+
+        <p>
+            The 'spark-core' module contains a class named <code>Startup</code> (in the <code>org.jivesoftware.launcher</code>
+            package) that can be used to start Spark. Find the class in the Project Explorer, open that class you will notice two hyperlinks above 
+            the main method, like in the picture above. You can click on <code>Run</code>, and the code will be compiled and executed VSCode wil start
```

Comment:
No comma after Run. Should be a dot after executed (end sentence).
will instead of wil